### ISSUE TITLE
TODO: V2 items for V3

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/ladybug-mixer#1]
- add about 6dB of additional amplification (make sure does not exceed ±2.5V output for typical input though) **DONE**
- change tantalums to higher voltage to avoid future explosions  **DONE**
- move pots to after the tone controls to reduce noise when pots at min position  **DONE**
- eliminate JST PH connectors (next version can hard-wire, if appropriate)  **DONE**
- switch to single OP AMP (probably pick MIC 1 op amp, which has 3dB lower noise)  **DONE**
- add some jumpers to tell MCU some config info (might need diodes or resistors here)  **DONE**
  - Remote vol control direction is reversed from normal
  - Remove vol control goes all the way to zero (vs not)
  - LM1971 is not present (can this be figured out without a jumper via I2C -- nope, this cannot)
  - etc
- eliminate test jumpers, replace with pads + surface trace  **DONE**
- finalize the name (LB-102 vs  LB-201 etc.) - yeah, it's now LB-102  **DONE**
- add rear panel pics to the cut sheet

**Comment by mpogue2** [comment]
Instead of jumpers, use 0 ohm resistors for test points.

**Comment by mpogue2** [comment]
Replace LM1971 with C129348, which costs about $2.60 ea.
These are out of stock, but can be pre-purchased. Delay is <20 working days.  Then I can have them auto-assembled into boards.  I can also "consign" parts, that is, I buy the parts and send to them.

**Comment by mpogue2** [comment]
**Pre-order**: https://jlcpcb.com/help/article/53-what-is-jlcpcb-parts-pre-order-service
**Consignment**: https://jlcpcb.com/help/article/245-how-to-consign-parts-to-jlcpcb

**Comment by mpogue2** [comment]
LM1971MX/NOPB (Arrow) = Q10+ $1.32 ea.  These are C129348 in JLCPCB's numbering system.
This looks like the best Global Sourcing deal.  Takes 8-15 days, plus 1-2 days for review.
I'm gonna switch to this part, and mark it Do Not Populate for these first prototypes, but this will make it easier to order them for automated assembly next time.

**Comment by mpogue2** [comment]
V3 boards (main board and XLR board) have been submitted to JLCPCB for fabrication.  Files have been posted to the repository.  Note that I could not do the manual op amp swap, because it generated an error and refused to populated the MCP6002s.  So, I went with the parts that I had in the schematic that are slightly more expensive, but should be identical.
The parts swap was only for testing (which is not needed for V3) and for cost reduction (minimal, so I gave up on it for V3).

V3 enclosure parts (5052-Al for top and bottom) have been submitted to Fabworks for fabrication. Files have been posted to the repository.

Closing this issue.
